File: Front/Eventos-App/src/app/_models/palestrante.ts
import {RedeSocial} from './redesocial';

export interface Palestrante {
	id: number;
	nome: string;
	miniCurriculo: string;
	imagemURL: string;
	telefone: string;
	email: string;
	redeSociais: RedeSocial[];
	//PalestranteEventos: PalestranteEvento[];
}

File: Back/src/Eventos.Domain/Palestrante.cs
using Eventos.Domain.Identity;

namespace Eventos.Domain;

public class Palestrante
{
	public int Id { get; set; }
	public string MiniCurriculo { get; set; }
	public int UsuarioId { get; set; }
	public Usuario Usuario { get; set; }
	
	public IEnumerable<RedeSocial> RedeSociais { get; set; }
	public IEnumerable<PalestranteEvento> PalestranteEventos { get; set; }
}